### ISSUE TITLE
Build 103 todo list

### ISSUE THREAD

**Issue by Mgamerz** [ME3Tweaks/ME3TweaksModManager#26]
The following items need to be completed before build 103 can be published.

**Localizations**
 - [x] Localizations for ~~French~~, ~~Polish~~
    - Russian: Done!
    - German: Done!
 - [x] Localize remaining unlocalized files
 - [x] Localize tips service strings
    - Russian: Done!
    - German: Done!

**Features**
 - [x] Handle ME3 logger - ensure only one version is installed
 - [x] Find place to put "Endorse ME3Tweaks Mod Manager ME1/ME2/ME3"
 - [x] Calculate required space to import mod as multiple crashes have been logged due to not enough disk space
 - [x] Calculate required space to install a mod as multiple crashes have been logged due to not enough disk space

**Testing**
 - [x] Test check/uncheck all mods
 - [x] Test NexusMods integration (need testers to do this as I cannot endorse my own mods)
    - Needs some fixes but not blocking. Will be done in Build 104.
 - [x] Test Alt Grouping
 - [x] Test Mass Effect INI Modder
 - [x] Test ASI Manager
 - [x] Test all UIs for localization strings being incorrect/wrong substitutions
 - [x] Test RCW mods (.me2mod)
   - Follow up testing: Additional bonus powers crashes M3 on install due to capitalization not correct in me2mod
 - [x] Test offline/bad network connection behavior

**Serverside**
 - [x] Update Tips Service on server
 - [x] Make server definitions for JAM

**Deployment**
 - [x] Soak test 103

**Comment by Mgamerz** [comment]
## Changelog (until release is finalized)
This is a feature release for M3. The main feature is localization support, which has proven to be a huge amount of work.

## New features
 - NexusMods integration allows you to endorse/unendorse mods directly from within Mod Manager, once you provide your API key. Only mods that have nexusmod's url's set or nexus codes assigned in their moddesc will work - other mods will need to be updated to link them to a nexusmods page
 - The groundwork for localizing the application into other languages has been implemented, however no localization are complete yet. Localiazations for Russian, German, Polish and French are  in progress. You can change languages in-app, but some (or none at all for early-work translations) will change. The app will be updated as my translators complete their work.
   - This was a very large amount of work. If you want to help contribute or report issues please file an issue on github or come to the Discord.
 - Mod Manager can now import and install .me2mod files, as well as handle archives containing .me2mod files (files for RCW Mod Manager).
 - Mass Effect Ini Modder has been merged into Mod Manager. Mass Effect Ini Modder (the standalone program version) has now been deprecated and is unsupported.
 - Developers can now specify default options that are checked for alternate files and alternate dlc's. 
 - Developers can now specify alternate "groups", where only one option in the group can be picked. At least 1 option in a group must be selected.
 - A new operation has been added that does nothing for both alternate DLC and alternate files. This is used in conjunction with alternate groups to provide a default option where the mod is essentially unchanged (the default version of the mod).
 - Implemented ASI Manager, so you can manage ASI mods for all three games.
 - LODs (alot texture settings) are enabled/disabled when the active target changes, to prevent incorrect LODs from being used by the active target if they don't match.
 - Implemented support for the `addfilesreadonlytargets` attribute in moddesc.ini files (used by EGM)



## Changed features
 - Non validated exe mods will no longer attempt to import into M3, as their contents may be parsed by the TPMI service even though they need special handling.
 - Some strings in the english version have changed after feedback from localization group
 - Some UI elements have been changed.
 - The size required to install or import a mod is now checked (if possible, depends on archive type) before the action is performed, plus a 5% buffer. If there is not enough space, the operation is aborted.
 - Lots more logging for troubleshooting

## Bugfixes
 - Fixed an issue that prevented mods from being able to use . as the moddir path. This is a workaround to support existing mod archives in the scene, such as ME1 Ultrawide.
 - Fixed application crash on theme change
 - Fix some status strings not showing due to strange threading issues
 - Implement alternate operation OP_INSTALL, which was curiously missing from 102.
 - Fix multi-dlc conditional alternates always applying even if they are not applicable (e.g. a mod was checking if multiple dlc existed, did not exist, any existed or non existed)
